src/test_c_kzg_4844.c

```diff
@@ -0,0 +1,81 @@
+/* Tests for myfunctions.c, using TinyTest. */
+
+#define UNIT_TESTS
+
+#include "tinytest.h"
+#include "blst.h"
+
+#include "c_kzg_4844.h"
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+
+KZGSettings s;
+
+void load_setup() {
+  FILE *fp;
+  C_KZG_RET ret;
+  fp = fopen("trusted_setup.txt", "rb");
```

Comment:
why `b`? This should be opened as text

Reply:
It was just a copy/paste of the code in this comment. Yeah the "b" should be removed.

* https://github.com/ConsenSys/teku/issues/6622#issuecomment-1356376309

Reply:
OK I'll do it